org.jcryptool.visual.rabin/src/org/jcryptool/visual/rabin/HandleFirstTab.java

```diff
@@ -0,0 +1,2075 @@
+package org.jcryptool.visual.rabin;
```

Comment:
This seems like it should be in the package ui
```suggestion
package org.jcryptool.visual.rabin.ui;
```